root/components/RelatedSeries.js

```diff
@@ -38,7 +37,18 @@ const RelatedSeries = ({seriesIds}: Props): React.MixedElement => {
       </h3>,
       <StaticRelationshipsDisplay
         relationships={
-          groupRelationships(series.relationships, targetEntityTypes)
+          groupRelationships(
+            series.relationships,
+            undefined,
+            (
+              relationship: RelationshipT,
+            ) => {
+              if (isNotSeriesPart(relationship)) {
+                return true;
+              }
+              return false;
+            },
+          )
```

Comment:
Unless we foresee extending this soon, you can also just pass the function directly AFAICT.
```suggestion
            isNotSeriesPart,
```